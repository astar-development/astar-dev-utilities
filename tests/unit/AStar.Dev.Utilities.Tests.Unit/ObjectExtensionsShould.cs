namespace AStar.Dev.Utilities;

public sealed class ObjectExtensionsShould
{
    [Fact]
    public void ContainTheToJsonMethodWhichReturnsTheExpectedString() =>
        new AnyClass()
           .ToJson()
           .ShouldMatchApproved();
}
