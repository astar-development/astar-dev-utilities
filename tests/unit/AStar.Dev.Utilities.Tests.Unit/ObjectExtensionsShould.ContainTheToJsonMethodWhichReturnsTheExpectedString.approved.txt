{
  "anyInt": 0,
  "anyString": ""
}